<?xml version="1.0"?>
<launch>
    <!-- obstacle_inflater_node -->
    <arg name="inflation_radius" default="0.15"/>
    <node pkg="a_star_ros" type="obstacle_inflater_node" name="obstacle_inflater" output="screen">
        <param name="inflation_radius" value="$(arg inflation_radius)"/>
    </node>

    <!-- a_star_node -->
    <arg name="global_frame" default="map"/>
    <arg name="debug_mode" default="false"/>
    <arg name="sleep_time" default="0.01"/>
    <arg name="map_topic" default="/map/inflated_map"/>
    <arg name="initial_pose_topic" default="/initialpose"/>
    <arg name="goal_pose_topic" default="/move_base_simple/goal"/>
    <node pkg="a_star_ros" type="a_star_node" name="a_star_planner" output="screen">
        <param name="debug_mode" value="$(arg debug_mode)"/>
        <param name="sleep_time" value="$(arg sleep_time)"/>
        <param name="global_frame" value="$(arg global_frame)"/>
        <remap from="/map" to="$(arg map_topic)"/>
        <remap from="/initialpose" to="$(arg initial_pose_topic)"/>
        <remap from="/move_base_simple/goal" to="$(arg goal_pose_topic)"/>
    </node>
</launch>
